src/Nri/Ui/Select/V9.elm

```diff
@@ -342,7 +342,7 @@ viewSelect config_ config =
         valueLookup =
             optionStringChoices
                 ++ groupStringChoices
-                |> List.map (\x -> ( x.id, x.value ))
+                |> List.map (\x -> ( x.strValue, x.value ))
```

Comment:
I'm noticing `groupStringChoices` looking at this code again.. Maybe we should add another test for when we group the options?